guides/micronaut-data-mongodb/src/test/groovy/example/micronaut/BaseMongoDataSpec.groovy

```diff
@@ -0,0 +1,68 @@
+package example.micronaut
+
+import example.micronaut.clients.CourseClient
+import example.micronaut.clients.StudentClient
+import io.micronaut.context.ApplicationContext
+import io.micronaut.runtime.EmbeddedApplication
+import org.testcontainers.containers.MongoDBContainer
+import org.testcontainers.utility.DockerImageName
+import spock.lang.AutoCleanup
+import spock.lang.Shared
+import spock.lang.Specification
+
+abstract class BaseMongoDataSpec extends Specification {
```

Comment:
This should implement `TestPropertyProvider`

See how it is done here https://guides.micronaut.io/latest/micronaut-mongodb-synchronous-gradle-java.html#test

Reply:
Cool thanks Graeme!  Fixed in https://github.com/micronaut-projects/micronaut-guides/pull/868/commits/a2fbfc6c160aecb5c5632bdac07910269b53a6f7